### ISSUE TITLE
Tracing root (Manual and Automatica tracing) 

### ISSUE THREAD

**Issue by BioresearchBO** [SmartRoot/SmartRoot#28]
Hello,

I am trying to trace the root. I follow every details that has been explained but nothing happened and the program does not trace anything (neither manual and automatic). What would be the problem and how to fix it?

Thank you
Armin

**Comment by guillaumelobet** [comment]
Hard to tell just like that. Did you open the image from the SR Explorer window? If you open an image from ImageJ directly, it will not work. 

Also, do you see any message in the log window?

**Comment by BioresearchBO** [comment]
> 
> 
> Hard to tell just like that. Did you open the image from the SR Explorer window? If you open an image from ImageJ directly, it will not work.
> 
> Also, do you see any message in the log window?

Yes, I did open the image using SR Explorer window but I wasn't able to trace the root. 

No, I did not receive anything in the log window.

**Comment by BioresearchBO** [comment]
> 
> 
> Hard to tell just like that. Did you open the image from the SR Explorer window? If you open an image from ImageJ directly, it will not work.
> 
> Also, do you see any message in the log window?

Another question. When I open SmartRoot using ImageJ, I get an error like the one in the photo that is attached:

![SmartRoot](https://user-images.githubusercontent.com/33976411/69976558-75a7e000-1529-11ea-9827-12927ea8fcba.png)

Could it be the problem for not measuring the root length?

Best,
Armin

**Comment by hibby8** [comment]
I am getting the same error messages in the log window as Armin and would be interested in finding a solution too! I wonder if it is related to the fact I downloaded the most recent version of java. 
My java console output also is producing this...
![image](https://user-images.githubusercontent.com/59769181/72205938-65681680-3480-11ea-9852-5ac54e1afef8.png)
